network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/session_slider_block.html

```diff
@@ -1,44 +1,47 @@
-        <div class="tw-relative tw-overflow-hidden {% if page.signup != None %} large:tw-mr-[calc(50%-50vw)] {% else %} tw-container{% endif %}">
-            <div class="swiper tw-px-6 tw-pb-24 tw-mx-0 tw-overflow-hidden tw-container" data-carousel>
+    <div class="tw-container tw-py-16">
+        <div class="tw-row">
+            <div class="tw-relative tw-overflow-hidden">
+                <div
+                    class="swiper tw-relative tw-pb-16 tw-mx-0 tw-overflow-hidden tw-container after:tw-hidden large:after:tw-block after:tw-content-[''] after:tw-bg-gradient-to-l after:tw-from-white tw-from-70% after:tw-to-transparent after:tw-h-[475px] after:tw-w-[40px] after:tw-top-[60px] after:tw-absolute after:tw-right-0 after:tw-z-10"
+                    data-carousel
+                    data-slides-per-view="1.2"
+                    data-desktop-slides-per-view="2.4"
+                >
 
-                {% block navigation %}
-                    <div class="tw-flex tw-justify-center tw-items-start tw-flex-col tw-mb-12 medium:tw-flex-row medium:tw-justify-between medium:tw-items-center">
-                        <h2 class="tw-h3-heading tw-font-semibold tw-mb-0 tw-w-full">{{ value.title }}</h2>
-                        {% if self.session_items|length > 2 %}
-                            {% include 'fragments/carousel/carousel_navigation.html' %}
-                        {% endif %}
-                    </div>
-                {% endblock %}
+                    {% block navigation %}
+                        <div class="tw-flex tw-justify-center tw-items-start tw-flex-col tw-mb-10 medium:tw-flex-row medium:tw-justify-between medium:tw-items-center">
+                            <h2 class="tw-h3-heading tw-mb-0 tw-w-full">{{ value.title }}</h2>
+                            {% if self.session_items|length > 2 %}
+                                {% include 'fragments/carousel/carousel_navigation.html' %}
+                            {% endif %}
+                        </div>
+                    {% endblock %}
 
-                {% block slides %}
-                    <div class="swiper-wrapper">
-                        {% for item in self.session_items %}
-                            {% image item.value.image fill-250x140 as img %}
-                            <div class="swiper-slide">
+                    {% block slides %}
+                        <div class="swiper-wrapper tw-items-stretch">
+                            {% for item in self.session_items %}
+                            {% image item.value.image fill-420x290 as img %}
+                            <div class="swiper-slide tw-h-auto">
                                 {% include 'fragments/session_card.html' with title=item.value.title author_subheading=item.value.author_subheading url=item.value.link.0.value.link_url image=img.url text=item.value.body animated_thumbnail=item.value.video.url %}
                             </div>
```

Comment:
I think this should be indented.

Reply:
Updated in https://github.com/MozillaFoundation/foundation.mozilla.org/pull/11588/commits/a4e15f54c9bae5663d4ba5d6936ebc33b611fad3